app/index.js

```diff
@@ -76,6 +75,7 @@ var FluxGenerator = yeoman.generators.Base.extend({
     this.composeWith('flux:dispatcher', {args: ['AppDispatcher']});
     this.composeWith('flux:store', {args: ['DataStore']});
     this.composeWith('flux:action', {args: ['DataActionCreators']});
+    this.composeWith('flux:component', {args: ['App']});
```

Comment:
This is good, thanks for clarifying. I will likely change the base generator to scaffold out the `controller-view` and then merge with your code. I can take it from here.

Thanks!